packages/workers-shared/utils/types.ts

```diff
@@ -6,8 +6,14 @@ const InternalConfigSchema = z.object({
 	debug: z.boolean().optional(),
 });
 
+const StaticRoutingSchema = z.object({
+	worker: z.array(z.string()),
+	asset: z.array(z.string()).optional(),
```

Comment:
```suggestion
	user_worker: z.array(z.string()),
	asset_worker: z.array(z.string()).optional(),
```

Just a pet peeve that the whole thing is a "Worker" (not just the user-provided script!)

Reply:
> this is currently user-facing configuration

Wait, is it? Okay, I need to go review your backend PR :) I thought this was all being lumped into `assets.run_worker_first` as an unlabelled array of `["positive", "!negative"]` matches. I figured that was how it was being passed to the API too, but I'll take a look.

Not blocking. We're inconsistent in a few places anyway. But I'll double-check this backend and get back to you :)

Reply:
yeah, the schema in wrangler.jsonc is indeed an unlabelled array. but the router worker will receive it as split and labelled, so we could either have EWC do so, Wrangler do so. Given that Wrangler already has to do so to support this in dev, @emily-shen thought it might be better to just let Wrangler do the splitting and only implement it in one place.

if the divergence between the Wrangler configuration and the public API feels like the wrong decision (and we're certainly guilty of unnecessarily doing so in the past) we can make the call to undo that and keep things those APIs in sync (even if that means duplicated logic)

Reply:
I vote dupe. I think Wrangler config should look the same as our API here, personally. But I don't put my foot down on it — happy to be overruled if I'm out-voted.

Reply:
i do think i agree, the more i think about it. ok, will fix this up, and that will make user_worker and asset_worker sensible keys here